src/ast/expression.rs

```diff
@@ -62,6 +97,20 @@ impl<'a> Expression<'a> {
           arguments,
           location: tokens.location(),
         }
+      } else if tokens.try_grammar(Grammar::ListClose).is_ok() {
+        // support multiple indexes ie a[0][1]
+        let mut expressions = Vec::new();
+        expressions.push(Expression::try_expression(tokens)?);
+
+        loop {
+          tokens.try_grammar(Grammar::ListOpen)?;
+
+          if tokens.try_grammar(Grammar::ListClose).is_err() {
+            break;
+          }
+          expressions.push(Expression::try_expression(tokens)?);
+        }
+        Expression::Index(identifier, expressions, tokens.location())
```

Comment:
Does it make more sense to have wrapped expressions here, or is a vec of indices okay? 🤔